### PR TITLE
update ci workflow

### PR DESCRIPTION
**Pull request by erfanium**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,11 +14,11 @@ jobs:
       - uses: actions/checkout@master
       - uses: denolib/setup-deno@master
         with:
-          deno-version: 1.1.1
+          deno-version: 1.1.2
```
Comment (disizali) [inline .github/workflows/ci.yml:17]: is something broken with 1.1.3?

Comment (disizali) [inline .github/workflows/ci.yml]: is something broken with 1.1.3?

Comment (erfanium) [reply .github/workflows/ci.yml]: No

**Review by disizali — APPROVED** [review]
LGTM, thank you!

**Pull request merged**